frontend/apps/marketing-storybook/applitools.config.cjs

```diff
@@ -6,6 +6,20 @@
 // Used in DevContainers, see `.devcontainer/frontend/Dockerfile`
 const isDocker = !!process.env.IS_DOCKER || !!process.env.CI;
 
+const variationMatrix = {
+  'code.org': {theme: 'code.org', sectionBackground: ['primary', 'dark']},
+  csforall: {theme: 'csforall', sectionBackground: ['primary']},
+};
+
+const variations = Object.entries(variationMatrix).flatMap(
+  ([, {theme, sectionBackground}]) =>
+    sectionBackground.map(bg => ({
+      queryParams: {
+        globals: `theme:${theme};sectionBackground:${bg}`,
```

Comment:
Did you confirm this works? The Code.org theme is using CSS variables and not the MUI palette.

Reply:
Seems to work - can you take a quick pass at the eyes report? You'll have to filter to `globals` = `theme:code.org;sectionBackground=dark`

<img width="731" height="242" alt="image" src="https://github.com/user-attachments/assets/e4c8380d-e95f-43eb-9b5f-74f94b9144d0" />